.gitignore

```diff
@@ -43,9 +43,20 @@ local_test/*
 !local_test/index.html
 local_test/js/*
 local_test/css/*
+# folder of projects to test 
+local_test/intesting
 
 # local test code
 localTestCode.txt
 
 # Apple files
-*.DS_Store
\ No newline at end of file
+*.DS_Store
+
+# textmate
+code_textmate
+code_textmate.tmproj
+
+package-lock.json
```

Comment:
I don't know how this file works, but the npm docs are saying it should be committed. https://docs.npmjs.com/files/package-lock.json